doc/distrib/NodeHelpFiles/_review/Revit.Elements.FamilyType.ByFamilyNameAndTypeName.md

```diff
@@ -0,0 +1,8 @@
+## In Depth
+Similar to `Revit.Elements.FamilyType.ByFamilyAndName`, `Revit.Elements.FamilyType.ByFamilyNameAndTypeName` returns the family type definition from the current document (if available). Rather than using a family definition, this node relies on string input for both values. If the family type is not available in the current document, a null value is returned.
```

Comment:
From a readability perspective, suggest rephrasing this as " Revit.Elements.FamilyType.ByFamilyNameAndTypeName returns the family type definition from the current document (if available). This is similar to `Revit.Elements.FamilyType.ByFamilyAndName`. But rather than using a family definition, this node relies on string input for both values. If the family type is not available in the current document, a null value is returned."